Libraries/LLM/Models.swift

```diff
@@ -110,6 +110,13 @@ public struct ModelConfiguration {
 }
 
 extension ModelConfiguration {
+    public static let smolLM_135M_4bit = ModelConfiguration(
+        id: "mlx-community/SmolLM-135M-Instruct-4bit",
+        defaultPrompt: "Tell me about the history of Spain."
+    ) {
+        prompt in
+        "<|im_start|>user\n\(prompt)<|im_end|>\n<|im_start|>assistant\n"
```

Comment:
Is this correct? I am having issues with the tokenizer for this model Ollama seems to have figured out:

https://github.com/ollama/ollama/issues/5731#issuecomment-2233890666

Reply:
That looks like the same template they are using.  What problem are you seeing?